### PR TITLE
Add a shortcut "forward" in the selected message options

### LINKED ISSUE #353: Add a shortcut "forward" in the selected message options
Presently to forward a message the Tchap-iOS user has to select the option share and select the Tchap application.

DINUM request to add a dedicated option "Transférer"  to prompt directly the room lists

### PR DESCRIPTION
**Pull request by gileluard**
fix #353

### CODE REVIEW

File: Tchap/Modules/Forward/ForwardCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+protocol ForwardCoordinatorDelegate: class {
+    /**
+     Called once the coordinator finished forward process
+     
+     @param coordinator caller of this method
+     @param roomID ID of the room the event has been forwarded to
+     @param error instance of the error if an error occured. Nil otherwise
+     */
+    func forwardCoordinator(_ coordinator: ForwardCoordinatorType, didForwardTo roomID: String, error: Error?)
```
Comment (giomfo) [inline Tchap/Modules/Forward/ForwardCoordinator.swift:27]: This method is not used (no delegate is provided to the ForwardCoordinator in the current implementation)
which means the error (if any) is not handled.

I think we can remove this method from the delegate for the moment, and handle the potential error at the ForwardCoordinator level, by adding an `AlertErrorPresenter` (or  an `ErrorPresenter`). Don't you think so?

**Comment by giomfo** [comment]
I did some tests, and observed the following problems:
- the pending invites are displayed in the rooms list -> we have to remove them
image
- when I selected the second room to transfer the selected content, I saw it move to the top of the list. This update of the list by taking into account the recent activity is unexpected. We should not update the room order 

Remark: I would prefer a header (navigation bar) similar to the current share extension header (without the tabs):
The white title would be "Transférer à"
image

**Comment by giomfo** [comment]
@gileluard Can you please review my 3 last commits on your branch? 
Please tell in comment of this PR if you approved it

**Review by giomfo — APPROVED** [review]

**Comment by gileluard** [comment]
LGTM

**Pull request merged**